node/executor/Cargo.toml

```
@@ -17,6 +17,7 @@ node-primitives = { path = "../primitives" }
 node-runtime = { path = "../runtime" }
 
 [dev-dependencies]
+test-client = { package = "substrate-generic-test-client", path = "../../core/test-client/generic/" }
```

Great work and I am already using this. One nit: this is confusing. `test-client` should always point to `path = "../../core/test-client/` (still exists all around the repo) and this should be named `generic-test-client`.

```suggestion
generic-test-client = { package = "substrate-generic-test-client", path = "../../core/test-client/generic/" }
```